testing/test_dsession.py

```diff
@@ -115,18 +115,18 @@ def test_schedule_batch_size(self, pytester: pytest.Pytester) -> None:
         # assert not sched.tests_finished
         sent1 = node1.sent
         sent2 = node2.sent
-        assert sent1 == [0, 2]
-        assert sent2 == [1, 3]
+        assert sent1 == [0, 1]
```

Comment:
This test was updated to match the new distribution method, could you please add a new test which checks that we are using round robin for smaller collections?

Reply:
There is already such test, called `test_schedule_fewer_than_two_tests_per_node`. Also, the update of this test is simply part of the revert too - it was modified in 09d79ace356ebf14d221cecd3fd8002d075a2034, and I'm just undoing that change.